docs/setup_guides/hpc.rst

```diff
@@ -181,7 +181,7 @@ Now, connect to the server using an ssh tunnel from your local machine
 
 ::
 
-    $ ssh -N -L 8877:r8i4n0:8877 -L 8878:r8i4n0:8787 username@cheyenne.ucar.edu
+    $ ssh -N -L 8877:r8i4n0:8877 -L 8787:r8i4n0:8787 username@cheyenne.ucar.edu
```

Comment:
Not 100% sure about this one, but 8787 instead of 8787 feels like a typo, since everywhere else 8878 is used.

Maybe a good idea would be to use ports that are more different for jupyter notebook and the dashboard port.